File: math/coq/dragonfly/patch-configure.ml
--- configure.ml.orig	2016-12-08 17:13:52.000000000 +0200
+++ configure.ml
@@ -809,7 +809,7 @@ let strip =
 (** * md5sum command *)
 
 let md5sum =
-  if List.mem arch ["Darwin"; "FreeBSD"; "OpenBSD"]
+  if List.mem arch ["Darwin"; "DragonFly"; "FreeBSD"; "OpenBSD"]
   then "md5 -q" else "md5sum"
 
 

File: devel/py-sip4/dragonfly/patch-configure.py
--- configure.py.orig	2021-02-26 15:28:41 UTC
+++ configure.py
@@ -141,6 +141,7 @@ def set_build_platform():
         "darwin":       "macx-g++",
         "dgux":         "dgux-g++",
         "freebsd":      "freebsd-g++",
+        "dragonfly":    "freebsd-g++",
         "gnu":          "hurd-g++",
         "hp-ux":        "hpux-acc",
         "irix":         "irix-cc",
